app/app/api/questions/route.js

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server';
+import knex from '@/app/api/knex';
+
+export async function GET() {
+  try {
+    const result = await knex('questions').orderByRaw('RANDOM()').limit(40);
+
+    return NextResponse.json(result);
+  } catch (error) {
+    console.error('Error fetching questions:', error.message);
+    return NextResponse.json(
+      { error: 'Internal Server Error' },
+      { status: 500 },
+    );
+  }
+}
```

Comment:
<img width="675" alt="image" src="https://github.com/user-attachments/assets/ca8059a5-97e7-48ce-8d7b-5599a2333218" />

Having answers in the request probably not the best way to perform a test :)